website/server/controllers/api-v3/pushNotifications.js

```diff
@@ -0,0 +1,108 @@
+import { authWithHeaders } from '../../middlewares/api-v3/auth';
+import _ from 'lodash';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../../libs/api-v3/errors';
+
+let api = {};
+
```

Comment:
Is there any case where we'd want to allow someone to remove a push device via an email link? That's the only case I can think for for using a GET.